agogosml_cli/cli/templates/cookiecutter.json

```diff
@@ -5,6 +5,7 @@
     "REPOSITORY_REPO": "agogosml",
     "PROJECT_NAME": "sample_agogosml_project",
     "PROJECT_NAME_SLUG": "sample_agogosml_project",
+    "AZURE_CONTAINER_REGISTRY": "agogoscr.azurecr.io",
```

Comment:
can't this be generic for any DNS name on a container registry?

Reply:
Acknowledged. I am going to promote this setting to a global setting and restrict this additional check to only azure settings. I added a user story to the backlog.